src/lib.rs

```diff
@@ -98,6 +98,10 @@ pub struct Message {
     /// The message itself.
     #[serde(rename = "m")]
     pub message: String,
+
+    /// adding extra args
+    #[serde(rename = "a")]
+    pub extra_args: HashMap<String, String>,
```

Comment:
Thinking about this more, I'm not sure this should be a `HashMap`. Can you explain why we need k-v pairs instead of just a `Vec<String>` for example?

Reply:
inspired by params ( validator crate ) => I do not imagine a param/arg without key => v .

imagine adding field: field name .
to colorize the errored form fields , you need the field name ..
this will gives better UX

we should not stick to axum-flash .. we should go further

Reply:
If you insist and are willing to add it , just let me know to change it to Vec<String>. it does not matter very much to me

Reply:
> imagine adding field: field name .

Can you go into a little more detail? When you say "field", do you mean a DOM tag, something else? I'm not clear on the difference between a field and field name.

Also if you could share the validator crate's approach, I'd be interested in looking at that too.

Reply:
Yes i mean a dom `<input name="foo"/>`

the arg would be 
` field : foo `

because user could add different args than the field 
like for instance 
` value: baz`
`color: red`

... etc 

for validator 

this is the link 
https://docs.rs/validator/0.17.0/validator/struct.ValidationError.html#structfield.params

Reply:
i would be happy to participate in your repos

tbh this is my first PR for me as a code

Reply:
Thanks for providing that link. I'm okay to use k-v pairs.

Reply:
Thx Max
This is very encouraging to me